src/main/java/com/dnd/namuiwiki/jwt/JwtFilter.java

```diff
@@ -0,0 +1,41 @@
+package com.dnd.namuiwiki.jwt;
+
+import com.dnd.namuiwiki.common.exception.ApplicationErrorException;
+import com.dnd.namuiwiki.common.exception.ApplicationErrorType;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jws;
+import jakarta.servlet.*;
+import jakarta.servlet.http.HttpServletRequest;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.util.AntPathMatcher;
+
+import java.io.IOException;
+import java.util.List;
+
+@Component
+@RequiredArgsConstructor
+public class JwtFilter implements Filter {
```

Comment:
```suggestion
public class JwtFilter implements OncePerRequestFilter {
```

인증은 한 번만 진행해도 되는 필터라 `OncePerRequestFilter`를 상속해도 좋을 것 같아요 ㅎㅎ